p2p/utils_test.go

```diff
@@ -0,0 +1,81 @@
+package p2p
+
+import (
+	"context"
+	"crypto/rand"
+	"strconv"
+	"testing"
+
+	"github.com/libp2p/go-libp2p-core/crypto"
+	"github.com/libp2p/go-libp2p-core/peer"
+	"github.com/stretchr/testify/require"
+	"go.uber.org/multierr"
+
+	"github.com/lazyledger/optimint/config"
+	"github.com/lazyledger/optimint/log"
+)
+
+type testNet []*Client
+
+func (tn testNet) Start(ctx context.Context) (err error) {
+	for i := range tn {
+		err = multierr.Append(err, tn[i].Start(ctx))
+	}
+	return
+}
+
+func (tn testNet) Close() (err error) {
+	for i := range tn {
+		err = multierr.Append(err, tn[i].Close())
+	}
+	return
+}
+
+func (tn testNet) WaitForDHT() {
+	for i := range tn {
+		<-tn[i].dht.RefreshRoutingTable()
+	}
+}
+
+func makeTestNetowork(t *testing.T, n int, conns map[int][]int, logger log.Logger) testNet {
```

Comment:
Yeah, this increases the readbility of the tests 👍🏼 

```suggestion
func makeTestNetwork(t *testing.T, n int, conns map[int][]int, logger log.Logger) testNet {
```
And the calling code as well.

Reply:
There should be :facepalm: reaction :D